experimental/packages/opentelemetry-instrumentation-grpc/test/helper.ts

```diff
@@ -289,10 +280,7 @@ export const runTests = (
     return result;
   };
 
-  async function startServer(
-    grpc: typeof grpcJs | typeof grpcNapi,
-    proto: any
-  ) {
+  async function startServer(grpc: typeof grpcJs, proto: any) {
```

Comment:
We should probably get rid of passing `grpc` to it as it's only grpc-js now anyway (same for the other places). :thinking:
I think this can be split into a follow up though.

Reply:
I have finished the cleanup on another branch, but it involves breaking changes. I will make a PR as soon as this one merges.